daemon/log.c

```diff
@@ -253,14 +253,14 @@ void log_message(MTC_S32 priority, PMTC_S8 fmt, ...)
         return;
     }
 
+    if (priority & MTC_LOG_PRIVATELOG && fpLogfile != NULL && privatelogflag)
```

Comment:
Would you not expect testing `fpLogfile != NULL` first to be the fastest evaluation?